### DISCUSSION TITLE
OpenHarmony platform adaptation Suggestions

### DISCUSSION THREAD

**Discussion by DSOE1024** [godotengine/godot-proposals#9554] [Platforms]
Hi, I would like to suggest that the Godot project adapt to openharmony, an emerging open source platform.
I've recently been running games on openharmony phones using the Unity Engine. The openharmony community is growing and more developers and users are using the platform, so it's worth starting to adapt
So far, both Unity and Cocos2D engines have been adapted to openharmony devices, and hopefully Godot will be too

https://gitee.com/openharmony

Love for Godot

**Comment by openaddr** [comment]
Looking forward to the compatibility with OpenHarmony

**Comment by AThousandShips** [comment]
@openaddr Please don't bump without contributing significant new information. Use the :+1: reaction button on the first post instead.
